functions/questions.js

```diff
@@ -0,0 +1,101 @@
+import * as firebase from 'firebase';
```

Comment:
It's a good idea. There are several files like this that could be refactored - enough so that I would recommend doing the work in a separate PR.

Reply:
> enough so that I would recommend doing the work in a separate PR.

Totally agree!

How does everyone feel about opening improvement issues where we can centralize discussion on these kinds of topics? @quasney24?